docs/platforms/react-native/session-replay/index.mdx

```diff
@@ -0,0 +1,138 @@
+
+```javascript
+import * as Sentry from '@sentry/react-native';
+
+Sentry.init({
+  dsn = "___PUBLIC_DSN___",
+  _experiments: {
+    replaysSessionSampleRate: 1.0,
+    replaysOnErrorSampleRate: 1.0,
+  },
+  integration: [
+    Sentry.mobileReplayIntegration({
+      maskAllText: true,
+      maskAllImages: true,
+    }),
+  ],
+});
+```
+
+<SignInNote />
+
+## Verify
+
+While you're testing, we recommend that you set <PlatformIdentifier name="replays-session-sample-rate" /> to `1.0`. This ensures that every user session will be sent to Sentry.
+
+Once testing is complete, **we recommend lowering this value in production**. We still recommend keeping <PlatformIdentifier name="replays-on-error-sample-rate" /> set to `1.0`.
+
+## Sampling
+
+Sampling allows you to control how much of your website's traffic will result in a Session Replay. There are two sample rates you can adjust to get the replays relevant to you:
+
+1. <PlatformIdentifier name="replays-session-sample-rate" /> - The sample rate for
+   replays that begin recording immediately and last the entirety of the user's session.
+2. <PlatformIdentifier name="replays-on-error-sample-rate" /> - The sample rate for
+   replays that are recorded when an error happens. This type of replay will record
+   up to a minute of events prior to the error and continue recording until the session
+   ends.
+
+Sampling begins as soon as a session starts. <PlatformIdentifier name="replays-session-sample-rate" /> is evaluated first. If it's sampled, the replay recording will begin. Otherwise, <PlatformIdentifier name="replays-on-error-sample-rate" /> is evaluated and if it's sampled, the integration will begin buffering the replay and will only upload it to Sentry if an error occurs. The remainder of the replay will behave similarly to a whole-session replay.
+
+## Privacy
+
+The SDK is recording and aggressively redacting all text and images. We plan to add fine controls for redacting, but in this version, we just allow either on or off. The default is on. Please don’t turn it off if you have sensitive data in your app. Before the Beta is complete, we'll give you the controls you need.
+
+<Note>
+
+If you encounter any data not being redacted with the default settings, please let us know through a [GitHub issue](https://github.com/getsentry/sentry-react-native/issues/new?assignees=&labels=Platform%3A+React-Native%2CType%3A+%F0%9F%AA%B2+Bug&projects=&template=BUG_REPORT.md).
+
+</Note>
+
+To disable redaction altogether (not to be used on applications with sensitive data):
+
+```javascript
+Sentry.mobileReplayIntegration({
+  maskAllText: true,
+  maskAllImages: true,
+  maskAllVectors: true,
+}),
+```
+
+## React Component Names
+
+Sentry helps you capture your React components and unlock additional insights in your application. You can set it up to use React component names instead of selectors.
+
+So instead of looking at selectors like this:
```

Comment:
```suggestion
Sentry helps you capture your React components and unlock additional insights in your application. You can set it up to use React component names.

So instead of looking at this:
```